packages/components/src/Table/Table.css

```diff
@@ -1,10 +1,12 @@
 .table {
   --border-color: var(--color-grey--lighter);
   --header-border-color: var(--color-blue);
-  --horizontal-inset: 0.5rem;
-
+  --horizontal-inset: var(--space-base);
   width: 100%;
+
+  border: solid 1px var(--border-color);
   border-collapse: collapse;
+  border-radius: var(--space-smallest);
```

Comment:
I can't get the radius working 👻 - I think something to do with `border-collapse`?